backend/src/packages/meditation/libs/validation-schemas/create-meditation-entry-request.validation-schema.ts

```diff
@@ -0,0 +1,40 @@
+import joi from 'joi';
+
+import { ContentType } from '#libs/enums/enums.js';
+
+import { MeditationEntryValidationMessage } from '../enums/enums.js';
+import { type MeditationEntryCreateRequestDto } from '../types/types.js';
+
+const createMeditationEntryRequest = joi.object<
```

Comment:
looks like it should be on backend side
```suggestion
const createMeditationEntry = joi.object<
```

Reply:
Done